common/source_model/src/main/scala/weco/catalogue/source_model/sierra/rules/SierraItemAccess.scala

```diff
@@ -388,6 +396,27 @@ object SierraItemAccess extends SierraQueryOps with Logging {
           )
         )
     }
+  } match {
+    case Some(accessCondition) =>
+      // We set the access terms on an item from the access terms on the bib.
+      // Note that we need to be careful about merging terms from Calm/Sierra harvested
+      // items, which is why we assert that we haven't set terms based on anything
+      // *except* 506 subfield ǂa.
+      require(accessCondition.terms.isEmpty)
+
+      val bibTerms = {
```

Comment:
So this means I will definitely need to get the bib data to construct `AccessCondition` correctly?